### ISSUE TITLE
Getting negative loss function in Autoregressive

### ISSUE THREAD

**Issue by Hakim777** [tensorflow/tensorflow#47323]
Hi

I'm using Autoregressive for density estimation, the same example that is written in this page:

https://www.tensorflow.org/probability/api_docs/python/tfp/bijectors/AutoregressiveNetwork?version=nightly

However, when I increase the dimension (event_shape) of the data that I want to find its density, from two to four for example, I start getting negative loss function!

Can anyone help me with this issue.
Thanks

**Comment by jvishnuvardhan** [comment]
@Hakim777 This is more related to TF probability. There is a separate repository for probability [here](https://github.com/tensorflow/probability/issues). 

Please post this issue in that repo and close here. Experts in that repo will resolve your issue. Thanks!
